### ISSUE TITLE
Make Firmware Update Mode more flexible

### ISSUE THREAD

**Issue by mkopec** [Dasharo/dasharo-issues#712]
### The problem you're addressing (if any)

Entering Firmware Update Mode currently results in a fully automatic firmware update process - Disabling appropriate security measures, booting DTS and automatically proceeding with firmware update. This is nice for deployment automation but it somewhat reduces the usability of FUM, for example, an user can't (easily) use FUM to flash their own binary, or enter DES credentials before updating.

### Describe the solution you'd like

Have some prompts to confirm if the user wants to proceed with the default action, or exit to menu and specify their own commands.

Or, add another option to disable locks for the duration of one boot, but without booting into anything or doing anything else automatically, to allow the user to do what they want.

### Where is the value to a user, and who might that user be?

Users who want to verify the binaries manually before flashing
Users switching from one FW branch to another
Advanced users who want to flash what they want

**Comment by macpijan** [comment]
@BeataZdunczyk @mkopec Sounds simple enough to plan to include in the next releases

**Comment by krystian-hebel** [comment]
> Maybe just allow cancellation (with a timeout) before proceeding with autoupdate in DTS

I'm surprised this isn't possible. What is this timeout for, then? So you can cut the power if you want to abort? In case of laptops, you would have to keep power button pressed for 4 seconds instead, which would leave just one second to read, understand and react before the update begins, possibly writing to flash.

**Comment by Firminator** [comment]
> Describe alternatives you've considered
> 
> Maybe just allow cancellation (with a timeout) before proceeding with autoupdate in DTS
 
I agree, automatic sounds good, but there should be a long enough timeout like Krystian stated, and an options like the ability to read the cloud-pass file from a flash drive the user is prompted to insert.
